### ISSUE TITLE
The phone locks when SIP and GSM calls arrive simultaneously 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sonuarya/csipsimple#818]
```
What steps will reproduce the problem?

1. I've got an asterisk that routes any incoming calls to me via SIP and PSTN 
simultaneously.
2. When csipsimple is active on my phone and I'm logged in to a SIP account, 
and a call arrives, the phone rings in parallel - SIP and GSM at the same time 
- two handsets are displayed in the notification bar.
3. There is no way to answer any of the two calls - the phone does not react to 
anything until both calls stop.

What is the expected output? What do you see instead?

In a situation when incoming SIP call is ringing AND a GSM call comes in - I'd 
like the SIP call to be just ignored (declined).

In a situation when a GSM call is ringing and a SIP call come in - I'd like the 
SIP call to be ignored.

What version of the product are you using? On what operating system?

nexus s 2.3.2, csipsimple nighly build 7xx-something.

```

Original issue reported on code.google.com by `bogo...@gmail.com` on 19 Mar 2011 at 8:08
- Merged into: #640
